templates/protractor/page_objects/homepage/components/NavigationComponent.js

```diff
@@ -0,0 +1,25 @@
+import Wait from '../../../helpers/WaitHelper';
+import Timeouts from '../../../constants/Timeouts';
+
+export default class NavigationComponent {
+  constructor() {
+    this.menuButton = element(by.css('.component.plain-html.menu-handler .component-content'));
```

Comment:
Let's keep things consistent and extract elements' selectors to a separate file, similar to what's done for https://github.com/Cognifide/cogE2E/blob/master/templates/puppeteer/page_objects/homepage/HomePage.po.js

Reply:
fixed